### ISSUE TITLE
FIA Pstgres Parsing Error

### ISSUE THREAD

**Issue by mshahriarinia** [weecology/retriever#476]
This seems to be due to the same thing as Issue #475
```
$ retriever install postgres FIA -u postgres -p postgres
=> Installing Forest Inventory and Analysis
Creating database FIA...
Creating table FIA.species...
Couldn't create table (syntax error at or near "!"
LINE 1: CREATE TABLE FIA.species (lthn_!doctype_html_public_w3c_dtd_...
                                       ^
). Trying to continue anyway.
INSERT INTO FIA.species (lthn_!doctype_html_public_w3c_dtd_xhtml_1_0_strict_en_http_www_w3_org_tr_xhtml1_dtd_xhtml1_strict_dtd_gthn) VALUES ('<html xmlns=\"http://www.w3.org/1999/xhtml\">');
syntax error at or near "!"
LINE 1: INSERT INTO FIA.species (lthn_!doctype_html_public_w3c_dtd_x...
                                      ^

Done!
``

**Comment by ethanwhite** [comment]
This is the same problem as #475 and is probably a cached data file from after they moved the data file, but before we updated our script to know where they'd moved it to. I suspect that `git reset all` and then rerunning the install command will fix it.

**Comment by mshahriarinia** [comment]
Solved as mentioned in #475. Thanks!
